source/NVDAObjects/IAccessible/sysListView32.py

```diff
@@ -333,7 +334,20 @@ def _getColumnContentRaw(self, index):
 		return buffer.value if buffer else None
 
 	def _getColumnContent(self, column):
-		return self._getColumnContentRaw(self.parent._columnOrderArray[column - 1])
+		targetColumn = self.parent._columnOrderArray[column - 1]
+		try:
+			return self._getColumnContentRaw(targetColumn)
+		except ArgumentError:
+			# In case that a 64-bit pointer to the text had been  returned /rjh
+			disp = DisplayModelTextInfo(self, 'all')
+			x,y,w,h = self._getColumnLocation(column)
+			start = disp._getClosestOffsetFromPoint(x,y)
+			end = disp._getClosestOffsetFromPoint(x+w,y+h)
+			# Include the whole word, even if it is white space /rjh
+			wordAtEnd = disp._getWordOffsets(end)

```

Comment:
I don't think that this will be necessary when you create a DisplayModelTextInfo limited to the rectangle of the text.